dataland-frontend/src/components/resources/frameworkDataSearch/p2p/P2pDataModel.ts

```diff
@@ -0,0 +1,1055 @@
+import { PathwaysToParisData, P2pSector } from "@clients/backend";
+import { Category } from "@/utils/GenericFrameworkTypes";
+
+export const p2pDataModel = [
+  {
+    name: "general",
+    label: "General",
+    color: "orange",
+    showIf: (): boolean => true,
+    subcategories: [
+      {
+        name: "general",
+        label: "General",
+        fields: [
+          {
+            name: "dataDate",
+            label: "Data Date",
+            description: "The date until when the information collected is valid",
+            component: "DateFormField",
+            required: true,
+            showIf: (): boolean => true,
+            validation: "required",
+          },
+          {
+            name: "sector",
+            label: "Sector",
+            description: "To which sector does the company belong?",
+            component: "InputTextFormField",
```

Comment:
Why is this not a Sector Dropdown?

Reply:
Because we did not work on the input types yet. We decided to do those when working on the upload page. But you are right this should and will be a dropdown component.

Reply:
Who added the frontend components in the data dictionary? If it was one of you, we should change it here.

Reply:
I set the component to "InputTextFormField" as it was the closest thing we got that compiled. It should, however, be changed during the upload form story at the latest.

Reply:
We have the `SingleSelectFormField` which provides a single select dropdown. And we already use it in the LksgDataModel.
<img width="302" alt="lksg_ref" src="https://github.com/d-fine/Dataland/assets/113936582/f98553af-9c15-4537-a5b3-728edfde18af">

Reply:
Created a sub task in the create upload page for p2p story to address this issue